pkg/sitewise/auth_test.go

```diff
@@ -0,0 +1,28 @@
+package sitewise
+
+import (
+	"context"
+	"testing"
+
+	"github.com/grafana/iot-sitewise-datasource/pkg/models"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSimpleAuth(t *testing.T) {
+	t.Skip()
+	settings := models.AWSSiteWiseDataSourceSetting{
+		EdgeAuthMode: "linux",
+		EdgeAuthUser: "username",
+		EdgeAuthPass: "password",
+	}
+
+	settings.Endpoint = "https://localhost:80"
```

Comment:
Does this test supposed to work? Maybe use httptest here?

```
ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
		fmt.Fprintln(w, "Hello, client")
}))
defer ts.Close()

settings.Endpoint = ts.URL
```

Reply:
yeah with the right credentials and cert it should work as expected - only meant for dev work hence skipped - will add comments for that

Reply:
using httptest is a great idea!  then we can actually test this function :)

Reply:
will do this in a separate PR